app.rb

```diff
@@ -2,6 +2,7 @@
 require './student'
 require './teacher'
 require './rental'
+require 'json'
```

Comment:
- [ ] When I delete the JSON files and try to start the app it crashes with the error below. Kindly ensure you handle the scenario when the JSON files do not exist - like creating the JSON files and initializing an empty array in them if they do not exist.

  ![image](https://user-images.githubusercontent.com/81468675/217275171-c609d52f-e59e-4cda-978e-1ae0af8ed573.png)